src/mmap_unix.rs

```diff
@@ -505,7 +580,13 @@ mod tests {
         let buf1 = [1u8, 2, 3, 4, 5];
 
         f.write_all(buf1.as_ref()).unwrap();
-        let r = MmapRegion::from_file(FileOffset::new(f, offset as u64), buf1.len()).unwrap();
+        let range = MmapRange::with_file(
+            buf1.len(),
+            Some(FileOffset::new(f, offset as u64)),
```

Comment:
`as ` conversions are considered bad practice because if later the type of e.g. `offset` changes and becomes larger, it would be silently truncated.

Reply:
I am not touching it for now, since it was already there. But I do understand what you are saying.